package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "author": "Evan Tahler <evantahler@gmail.com>",
+  "name": "ah-swagger-plugin",
+  "version": "0.0.1",
+  "description": "A swagger plugin for the actionhero servers",
+  "main": "index.js",
+  "scripts": {
+    "test": "jest"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/actionhero/ah-swagger-plugin.git"
+  },
+  "keywords": [
+    "actionhero",
+    "swagger",
+    "plugins"
+  ],
+  "engines": {
+    "node": ">=8.0.0"
+  },
+  "peerDependencies": {
+    "actionhero": ">=18.0.0"
+  },
+  "license": "Apache-2.0",
```

Comment:
If you are picking Apache 2, please also include a LENIENCE.TXT in the project

Reply:
Alright